x/stakeibc/keeper/msg_server_redeem_stake_test.go

```diff
@@ -104,9 +104,19 @@ func (s *KeeperTestSuite) TestRedeemStake_Successful() {
 	user := tc.user
 	redeemAmount := msg.Amount
 
-	// get the initial unbonding amount *before* calling liquid stake, so we can use it to calc expected vs actual in diff space
-	_, err := s.GetMsgServer().RedeemStake(sdk.WrapSDKContext(s.Ctx), &msg)
-	s.Require().NoError(err)
+	// Split the message amount in 2, and call redeem stake twice (each with half the amount)
+	// This will check that the same user can redeem multiple times
```

Comment:
nit: should we also change the RR between the two `RedeemStake`s to validate we handle changing RRs correctly?

Reply:
decision: use non 1.0 RR, but keep RR consistent between messages